Makefile

```diff
@@ -9,7 +9,7 @@ install:
 
 test:
 	@ echo -e "$(BUILD_PRINT)Running the tests"
-	@ pytest || true
+	@ pytest
 
 lint:
```

Comment:
I wonder how much will we use this listing, as any way a part of it is done by PyCharm.

But there are way too many warnings to have a perfect code. We might eventually want to select a sub-set of checkers that this linter can do. But probably not all.